README.md

```diff
@@ -119,7 +119,7 @@ You can run the Operator locally against a remote namespace. The name of the nam
 ```sh
 $ make setup/gomod
 $ make cluster/install/local
-$ make code/run
+$ LOCAL_INSTALL="True" make code/run
```

Comment:
I think it's safer to just hardcode it to `LOCAL_INSTALL="True"` in the Makefile (in code/run)

Reply:
When the operator is running in a cluster, it is executed in a container.
Usually the command defined by `ENTRYPOINT` declaration in Dockerfile is executed.
But through pod spec it is possible to override the ENTRYPOINT.